app/Pasteboard.m

```diff
@@ -0,0 +1,258 @@
+#include <string.h>
+#import <UIKit/UIKit.h>
+#include "fs/poll.h"
+#include "fs/dyndev.h"
+#include "kernel/errno.h"
+#include "debug.h"
+
+// Prepare for fd separation
+#define fd_priv(fd) fd->clipboard
+
+#if 1
+#define CLIP_DEBUG_PRINTK(fmt, ...) printk(fmt, ##__VA_ARGS__)
```

Comment:
I'd prefer for debug logging which is mainly useful during writing the code to be removed before merging. If you'd rather not do that, use TRACE from debug.h, which can already be toggled with a compile time flag.

Reply:
@tbodt Yup, I was planning to remove it once I was completely sure it's finished. Shouldn't have removed `WIP` tag yet :)

Reply:
FYI GitHub lets you mark a PR as "draft", which shows it in grey and prevents it from being merged